src/writeData/subscriptions/components/JsonParsingForm.tsx

```diff
@@ -141,7 +143,7 @@ const JsonParsingForm: FC<Props> = ({formContent, updateForm}) => {
                   )
                 }
                 status={status}
-                maxLength={16}
+                maxLength={56}
```

Comment:
just curious, why was 56 chars chosen?

Reply:
this was already in the string parsing one so i was just making it longer based on what gary said this morning but open to suggestions

Reply:
Looking at the db schema, the Json ones can be unlimited in length. The string ones have a max length of 255 chars. So I think we should probably mirror that here too?